esphome/components/mhz19/mhz19.cpp

```diff
@@ -51,20 +98,49 @@ void MHZ19Component::update() {
 }
 
 bool MHZ19Component::mhz19_write_command_(const uint8_t *command, uint8_t *response) {
-  this->flush();
-  this->write_array(command, MHZ19_REQUEST_LENGTH);
-  this->write_byte(mhz19_checksum(command));
+  bool ret;
+  int rx_error = 0;
+
+  do {
+    ESP_LOGD(TAG, "cmd [%s]", dump_data_buf(command));
```

Comment:
```suggestion
    ESP_LOGVV(TAG, "cmd [%s]", dump_data_buf(command));
```

definitely not a debug-level message